### PR TITLE
Feature/init video url async

### PR DESCRIPTION
**Pull request by dogo**
When you try to load a big remote video a huge "delay" occurs when the AVPlayer URL is initialising. (Sync operation)

Teste Video file :     https://dl.dropboxusercontent.com/u/12595498/Batman%20feira%20da%20fruta%20%28remasterizado%29-SD.mp4

Current master code behaviour
https://dl.dropboxusercontent.com/u/12595498/before.mov

This PR code behaviour
https://dl.dropboxusercontent.com/u/12595498/after.mov

### CODE REVIEW

**Comment by autresphere** [comment]
Interesting proposal, but it would be great if the activityInteractor was not appearing at all when the movie is local.

**Comment by dogo** [comment]
PR updated @autresphere. Let me know if it's  okay.

Thanks

File: ASMediaFocusManager/ASMediaFocusController.m [hunk 1]
```diff
@@ -225,19 +226,25 @@ - (void)showPlayerWithURL:(NSURL *)url
...
+        [self.activityIndicator startAnimating];
+    }
     
     dispatch_async(dispatch_get_main_queue(), ^{
         self.player = [[AVPlayer alloc] initWithURL:url];
         ((PlayerView *)self.playerView).player = self.player;
         [self.player.currentItem addObserver:self forKeyPath:@"presentationSize" options:NSKeyValueObservingOptionNew context:nil];
         [self layoutControlView];
-        [activityIndicator stopAnimating];
+        if (self.activityIndicator != nil)
```
Comment (autresphere) [inline ASMediaFocusManager/ASMediaFocusController.m]: Don't need to check for non nil, as if it's nil, the following call will just do nothing.

**Comment by dogo** [comment]
PR updated with proposed changes. 
Thanks @autresphere

**Pull request merged**